ia-wishlist-bot/adding_books_via_ia_wishlist.py

```diff
@@ -0,0 +1,51 @@
+"""
+Python Script to add books to Open Library using the Open Library Client
+
+Input: 'wish_list_march_2018.ndjson'
+* Parameters for the wishlist include "isbn10", "language", "author", "isbn13", "title" , "oclc", "date"
+
+Output: Adding new works to Open Library 
+"""
+
+# Using the Open Library Client
+from olclient.openlibrary import OpenLibrary
+import olclient.common as common
+import ndjson
+
+# Creating an object of the Open Library Client
+ol = OpenLibrary()
+
+# Use the Wishlist ndjson file
+with open('wish_list_march_2018.ndjson') as f:
+	data = ndjson.load(f)
+
+# Iterates over the size of the data
+for i in range(len(data)):
+
+	new_book = data[i]
+
+	# Data of the book
+	title = new_book['title'] if new_book['title'] else u''
```

Comment:
these may all throw KeyErrors. I think what you want is:
`title = new_book.get('title', u'')`

This will fetch the title if such a key exists, or fallback to `u''`